<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>478</width>
      <height>308</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="calendarSettingLabel">
       <property name="text">
        <string>Calendar settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCalendarWidget" name="calendarWidget"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addTrainiingButton">
            <property name="text">
             <string>Add training</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteTrainingButton">
            <property name="text">
             <string>Delete training</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editTrainingButton">
            <property name="text">
             <string>Edit training</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="viewTrainingButton">
            <property name="text">
             <string>View training</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveToPdfButton">
            <property name="text">
             <string>Save to PDF</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="StatsButton">
            <property name="text">
             <string>Stats</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>203</width>
      <height>140</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Training time settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="timeOfDayLabel">
            <property name="text">
             <string>Time of day: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="timeOfDayComboBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="timeLabel">
            <property name="text">
             <string>Time:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="trainingTimeTimeEdit">
            <property name="focusPolicy">
             <enum>Qt::WheelFocus</enum>
            </property>
            <property name="dateTime">
             <datetime>
              <hour>12</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2000</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
            <property name="timeSpec">
             <enum>Qt::LocalTime</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="durationLabel">
            <property name="text">
             <string>Duration(minutes):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="durationSpinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>332</y>
      <width>568</width>
      <height>284</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Exercises list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="typeOfTrainingLabel">
                <property name="text">
                 <string>Type of training: </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="typeOfTrainingComboBox"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QListView" name="exercisesListView"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Selected exercises</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QListWidget" name="selectedExercisesList"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QPushButton" name="deleteExerciseButton">
              <property name="text">
               <string>Delete one</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteAllExercisesButton">
              <property name="text">
               <string>Delete all</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>192</y>
      <width>358</width>
      <height>106</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="playersSettingLabel">
       <property name="text">
        <string>Players settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Players:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="playersSpinBox"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>218</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="picAPlayerLabel">
            <property name="text">
             <string>Pic a player:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="picPlayerComboBox">
            <property name="minimumSize">
             <size>
              <width>161</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addPlayerButton">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as_PDF"/>
    <addaction name="actionPreview"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionAdd_anew_player"/>
    <addaction name="actionDelete_player"/>
    <addaction name="actionEdit_player"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as_PDF"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_anew_player"/>
   <addaction name="actionDelete_player"/>
   <addaction name="actionEdit_player"/>
   <addaction name="separator"/>
   <addaction name="actionPreview"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/save_doc.png</normaloff>:/images/save_doc.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as_PDF">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/pdf.png</normaloff>:/images/pdf.png</iconset>
   </property>
   <property name="text">
    <string>Save as PDF</string>
   </property>
  </action>
  <action name="actionAdd_anew_player">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/add_user.png</normaloff>:/images/add_user.png</iconset>
   </property>
   <property name="text">
    <string>Add new player</string>
   </property>
  </action>
  <action name="actionDelete_player">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/delete_user.png</normaloff>:/images/delete_user.png</iconset>
   </property>
   <property name="text">
    <string>Delete player</string>
   </property>
  </action>
  <action name="actionEdit_player">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/edit_user.png</normaloff>:/images/edit_user.png</iconset>
   </property>
   <property name="text">
    <string>Edit player</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/preview.png</normaloff>:/images/preview.png</iconset>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
